modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheVersionGenerationWithCacheStorageTest.java

```diff
@@ -210,7 +210,7 @@ private void checkGridCacheVersionsGenerationOrder(Consumer<IgniteEx> actions, S
                     latch.await();
                 }
                 catch (InterruptedException e) {
-                    e.printStackTrace();
+                    throw new RuntimeException(e);
```

Comment:
Probably it works, but it's too hardcore to kill node with FH it's being stopped.
Thread.currentThread().interrupt(); would do the trick.